src/GraphQl/Type/FieldsBuilder.php

```diff
@@ -230,8 +230,9 @@ private function getResourceFieldConfiguration(string $resourceClass, ResourceMe
                 $className = $this->typeBuilder->isCollection($type) && ($collectionValueType = $type->getCollectionValueType()) ? $collectionValueType->getClassName() : $type->getClassName();
             }
 
-            $args = [];
```

Comment:
Why the removal?

Reply:
Missed/deleted that part copying the changes from my local version 🙈 should be fixed now 🙂